Implement skin.hide_download_controls property (#8529)
cBioPortal/cbioportal: docs/portal.properties-Reference.md

skin.right_logo=
```

> Prevent users from saving data by removing all Download tabs and download and copy-to-clipboard controls.

Reviewer: ```suggestion
Make it harder for users to download data by removing all Download tabs and download and copy-to-clipboard controls.
```
Not sure if preventing is really possible
